### ISSUE TITLE
Doc should mention libapache2-mod-wsgi-py3 dependency

### ISSUE THREAD

**Issue by regisrob** [loris-imageserver/loris#534]
... as it seems to be required to run Loris > 2.3.3 with Apache. Installing `libapache2-mod-wsgi-py3` solved #533.

So [apache.md](https://github.com/loris-imageserver/loris/blob/90643bace94326ee75a871f6f34fb48f975e4824/doc/apache.md#L11) and [dependencies.md](https://github.com/loris-imageserver/loris/blob/90643bace94326ee75a871f6f34fb48f975e4824/doc/dependencies.md#L38) should be updated to mention this dependency, depending on the version of Loris (but I guess `libapache2-mod-wsgi-py3` should be the default and recommended one).

I'd be happy to make a PR for that if it is a valid assertion.

**Comment by bcail** [comment]
yes, Loris doesn't support python 2 anymore, so all docs should reference python 3 packages (unless there's a note with the python 2 package that it only works for old versions of loris). A PR would be welcome.
